Sources/SwifterSwift/MapKit/MKMapViewExtensions.swift

```diff
@@ -44,7 +44,25 @@ public extension MKMapView {
 
         return annotationView
     }
-
+    
+    /// SwifterSwift: Zooms in on multiple mapView coordinates.
+    ///
+    /// - Parameters:
+    ///   - coordinates: Gets the array of type CLLocationCoordinate2D.
+    ///   - meter: Takes the type of Double zoom.
+    ///   - insets: Gives space to the CGFloat type screen.
+    ///   - animated: The animation control takes the Boolean value.
+    @available(iOS 9.0, *)
+    func multipleCoordinateZoom(for coordinates: [CLLocationCoordinate2D], meter: Double, insets: CGFloat, animated: Bool) {
```

Comment:
This signature feels awkward to me for a few reasons:
- It declares the cardinality of the parameters in its name.  It might be better named something like `zoom(to coordinates:...)`
- The `meter` parameter is used only when `coordinates.count == 1` and the `insets` parameter is used only when `coordinates.count > 1`

Reply:
Totally agree, also it didn't occur to me why insets param is a Float and not actually an `UIEdgeInsets`

Reply:
Agreed, insets should be a `UIEdgeInsets` value, not a `CGFloat`.

Reply:
I'll edit and send commit again :)